styles/projects-list.css

```diff
@@ -113,3 +115,62 @@ textarea.input {
   overflow: hidden;
   padding-right: 15px;
 }
+
+/* Blog Post styles */
+.post-image {
```

Comment:
**suggestion:** add explicit dimensions to these images to prevent [layout shift](https://web.dev/cls/). I think 400x225 should work well here.

Reply:
Done!